src/Charts/ChartListParams.php

```diff
@@ -79,6 +90,10 @@ public function toArray()
             $result["expand"] = "events";
         }
 
+        if ($this->validation != false) {
```

Comment:
Does this also work when `null` is passed in? In that case, no validation should happen.

Reply:
It should now ;)